reminders/migrations/0001_initial.py

```diff
@@ -25,9 +25,15 @@ class Migration(migrations.Migration):
                     validators=[reminders.class_models.reminder_models.validate_date])
                  ),
                 ('messages', models.TextField(blank=True, null=True)),
-                ('method', models.CharField(
-                    choices=[('ema', 'Email'), ('web', 'Website'), ('wae', 'Website and Email')],
-                    default='web', max_length=3)
+                ('method',
+                    models.CharField(
+                        choices=[
+                            ('ema', 'Email'), ('web', 'Website'), ('wae', 'Website and Email'),
+                            ('run', 'Run algorithm'), ('evt', 'Voting timeout')
+                        ],
+                        default='web',
+                        max_length=3
```

Comment:
I think you need to update also the max_length like:
```suggestion
                        max_length=5
```

Reply:
The `max_length` refers to the number of characters in the first names in the tuples and not to the number of tuples.